feeder/ethereum/README.md

```diff
@@ -0,0 +1,24 @@
+# Ethereum Witness Feeder
+
+## To run
+
+- cd into this directory (eg `feeder/ethereum`)
+- make sure your local environment matches the `.env.example` file in this directory
```

Comment:
Does this mean I should create a `.env` file in this directory based on the `.env.example` file, or `export` my own set of environment vars in a shell?

Reply:
Yup. Added some more details to hopefully make that more clear.